src/main/java/core/basesyntax/figure/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.figure;
+
+public class Square extends AbstractFigure implements Figure {
+    private double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square, area: "
```

Comment:
Don't use System.out.println() in the draw method. Consider using a return statement and let the caller of the method handle the output to the console.